### DISCUSSION TITLE
tailwindcss is a true gamechanger

### DISCUSSION THREAD

**Discussion by aryaniyaps** [tailwindlabs/tailwindcss#4593] [Kind words]
I was writing custom css classes traditionally with react before, because I wanted to have my own touch in my projects and not just stick with material UI or any other library which have their "own" styles. Also, these libraries were not easy to work with for me.

I got to know about tailwindcss recently and I really like it.
I can extend the framework for my needs and also, the Just In Time compiler was life changing.

Thanks for maintaining this great library!

**Comment by simonswiss** [comment]
That's awesome to hear!

Yeah, I think one of the unique value proposition of Tailwind is that it doesn't come with a predefined look/design (outside of colors, spacing, shadows, etc).

The fact that you can build very closely almost any UI with the defaults and no custom CSS is a pretty strong testament of that ✨
